docs/manual/kinds/sros.md

```diff
@@ -0,0 +1,686 @@
+
+## Managing Nokia SR OS nodes
+
+A Nokia SR OS node launched with containerlab can be managed via the following interfaces:
+
+/// tab | CLI
+Connect to the SR OS CLI:
+
+```bash
+ssh admin@<node-name/node-mgmt-address>
+```
+
+///
+/// tab | NETCONF
+NETCONF server is running over port 830
+
+```bash
+ssh admin@<node-name> -p 830 -s netconf
+```
+
+///
+/// tab | gNMI
+Using the best-in-class [gnmic](https://gnmic.openconfig.net) gNMI client as an example:
+
+```bash
+gnmic -a <container-name/node-mgmt-address> --insecure \
+-u admin -p NokiaSros1! \
+capabilities
+```
+
+///
+
+### Credentials
+
+Admin user credentials for the Nokia SR OS launched by Containerlab are:
+
+* username: `admin`
+* password: `NokiaSros1!`
+
+> Note: the admin password is changed by Containerlab from the default `admin:admin` combination.
+
+### Logs
+
+Logs can be retrieved with standard log commands for the given container runtime:
+
+```bash
+$ docker logs -f clab-sros-sr-sim1
+NOKIA_SROS_CHASSIS=SR-1
+NOKIA_SROS_SYSTEM_BASE_MAC=1c:30:00:00:00:00
+
+** Container version: 25.7.R1 (Built on Mon July 16 00:00:00 UTC 2025)
+
+
+** using configuration file: /etc/opt/nokia/sros.cfg
+mgmtIf=eth0
+ifDynamic=1
+cfDirs=/home/sros/chroot/cf1:;/home/sros/chroot/cf2:;/home/sros/chroot/cf3:
+logDir=/var/opt/nokia/log
+bootString=TIMOS: slot=a chassis=sr-1 card=cpm-1 mda/1=me6-100gb-qsfp28 mda/2=me12-100gb-qsfp28 features=2048
+cpuCount=2
+** linking /home/sros/chroot/cf1: to /home/sros/cf1:
+** linking /home/sros/chroot/cf2: to /home/sros/cf2:
+** linking /home/sros/chroot/cf3: to /home/sros/cf3:
+
+Looking for cf3:/bof.cfg ... OK, reading
+<SNIP>
+
+```
+
+## Interface naming
+
+You can use [interfaces names](../topo-def-file.md#interface-naming) in the topology file like they appear in -{{ kind_display_name }}-.
+
+The interface naming convention inside the SR OS command line is typically: `L/xX/M/cC/P`, `L/xX/M/P`, `L/M/cC/P` or `L/M/P` where:
+
+* `L` : line card number
+* `X` : xiom number (when present)
+* `M` : MDA position
+* `C` : cage or connector number
+* `P` : breakout port inside the port connector.
+
+SR OS interface names can be directly used in containerlab topology files, in the `links` section of the topology file.  
+Inside the SR OS container, these interfaces are mapped so they follow Linux interface name conventions: `eL-xX-M-cC-P`, `eL-xX-M-P`, `eL-M-cC-P`, or `eL-M-P`. Note that the prefix `e` is added at the beginning of the port, and the forward slash `/` is replaced with a hyphen `-`.
+
+Here is an example on how interfaces inside Nokia SR-SIM container are mapped to the cards, mdas, and connectors:
+
+```
+e1-2-3       -> card 1, mda 2, port 3
+e1-2-c3-1    -> card 1, mda 2, connector 3, port 1
+e2-2-c3-4    -> card 2, mda 2, connector 3, port 4
+e1-x2-3-4    -> card 1, xiom 2, mda 3, port 4
+e1-x2-3-c4-5 -> card 1, xiom 2, mda 3, connector 4, port 5
+```
+
+Thus, in the containerlab topology links' section you would use the following format:
```

Comment:
I am not a big fan of the wording of this -- it implies to me that you _need to_ convert the interface names before you can use them in Containerlab.

Since this is the first time the user comes across links in an SR-SIM topology in this doc, it can be a bit confusing; maybe this example should be put later in the file (or an example should be put before with regular SR OS interface naming?). Just for UX's sake.

Reply:
Reads much better, thank you!